frontend/app/src/components/UserInfo/UserInfo.tsx

```diff
@@ -0,0 +1,23 @@
+import { User } from "@/types/user";
```

Comment:
oops i forgot to export this component in the 
`frontend/app/src/components/UserInfo/index.ts` can help me to export it there